### DISCUSSION TITLE
When will Play 2.9 be released?

### DISCUSSION THREAD

**Discussion by devinjz** [playframework/playframework#11714] [Q&A] [answered]
When will Play 2.9 be released?

**Comment by HenningHeinz** [comment]
Hello, 

you can have a look at the Milestone page
https://github.com/playframework/playframework/milestone/105

> Due by March 31, 2023

An official release date imho has not been announced yet but it seems we are getting closer.
Cheers

Henning

**Reply by mkurz** [reply]
Release is planned after Scala 3 / Scala 2.13.11 / 2.12.18 is released, for more details see https://github.com/playframework/playframework/issues/11260#issuecomment-1495681382

**Reply by mkurz** [reply]
Play 2.9 release candidate available: https://github.com/playframework/playframework/releases/tag/2.9.0-RC2

**Reply by Squeng** [reply]
Thanks again and kudos, @mkurz !

It's very nice when
![grafik](https://github.com/playframework/playframework/assets/7399173/8031c213-3731-40c4-94a3-f3a6c28b0099)
turns into
![grafik](https://github.com/playframework/playframework/assets/7399173/9971d704-767e-41b8-b043-aacc60957f19)
just by updating Play.

(See https://github.com/Squoss/Fixadat/security/dependabot for details if you're interested.)

**Comment by realjax** [comment]
So all mentioned versions (Scala 3.3.0 2.13.11 and 2.12.18 ) are released  Any news on release date of 2.9?

**Reply by mkurz** [reply]
Sorry, I had a bit of a family emergency the last two weeks which took all my capacity, I will try to finally get things going over the weekend and next week. I am absolutely sorry for the delay.

**Reply by He-Pin** [reply]
> Sorry, I had a bit of a family emergency the last two weeks which took all my capacity, I will try to finally get things going over the weekend and next week. I am absolutely sorry for the delay.

Take care

**Reply by mkurz** [reply]
Play 2.9 release candidate available: https://github.com/playframework/playframework/releases/tag/2.9.0-RC2

**Reply by yle-heikkij** [reply]
Any news?

**Reply by mkurz** [reply]
Play 2.9.0 and 3.0.0 are available on Maven central already, you can use them now. Announcement will follow (looks like Monday).

**Reply by raul782** [reply]
This is the best news I've heard this year.

**Comment by mkurz** [comment, marked as answer]
Play 2.9 and 3.0 have been released now: https://twitter.com/playframework/status/1721576472047911007
